src/lib.rs

```diff
@@ -68,15 +68,13 @@ pub fn leftpad_with<'a, S>(string: S, len: usize, pad_char: char) -> Cow<'a, str
 /// # Examples
 ///
 /// ```
-/// use left_pad::leftpad;
+/// use left_pad::{leftpad,leftpad_with};
```

Comment:
Wow, wonder why it worked for me then.. whatevs, I don't really mind whether the imports are combined or not, was just curious.